src/iconsManifest/supportedExtensions.ts

```diff
@@ -4861,6 +4887,12 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
+    {
+      icon: 'vscode-insiders',
+      extensions: ['code-workspace'],
```

Comment:
Shouldn't we have here also `code-snipperts`?

Reply:
Yes. I’ve updated it to match the other vscode icon configurations.